### PR TITLE
Remove redundant overrides

### PR DESCRIPTION
**Pull request by hvitved**
Created using an automatic patch based on https://github.com/github/codeql/pull/10102.

### CODE REVIEW

File: ql/ql/test/callgraph/Foo.qll [hunk 1]
```diff
@@ -10,9 +10,7 @@ class Foo extends AstNode {
   predicate baz() { bar() }
 }
 
-class Sub extends Foo {
```
Comment (erik-krogh) [inline ql/ql/test/callgraph/Foo.qll:13]: You should revert the change to this file.  
It's a QL-for-QL test file. 

(Or you could make that override non-redundant, and update the expected test output).

**Review by erik-krogh — APPROVED** [review]
QL / JS 👍

**Review by MathiasVP — APPROVED** [review]

**Review by tausbn — APPROVED** [review]
Python :+1:

**Review by alexrford — APPROVED** [review]
Ruby 👍
